src/App.js

```diff
@@ -1,19 +1,38 @@
 import React, { useState, useEffect } from 'react';
 import { getAllData } from './util/index';
+import { AuthProvider } from 'react-auth-kit';
 import { createServer } from 'miragejs';
+import UserRegistration from './components/UserRegistration';
+import Contact from './components/Contact';
+import Home from './components/Home';
+import Logout from './components/Logout';
+import { Routes, Route } from 'react-router-dom';
 
 createServer({
   routes() {
     this.get('http://localhost:8000/api/v1', { data: 'This is a music app' }),
-      this.passthrough('http://localhost:8000/*'); // everything else will try to actually call the backend
+      // route to submit user registration form
+      this.post(
+        'http://localhost:8000/api/v1/auth/register',
+        (schema, request) => {
+          let attrs = JSON.parse(request.requestBody);
+          console.log(attrs);
+        }
+      );
+    //route to logout user
+    this.post('http://localhost:8000/api/v1/auth/logout', (schema, request) => {
+      let attrs = JSON.parse(request.requestBody);
+      console.log(attrs);
+    });
```

Comment:
After you merge your Register page PR, then this will have to move to the new `mirageServer` file